colossalai/gptq/gptq_utils/gptq.py

```diff
@@ -0,0 +1,236 @@
+import math
+import time
+
+import torch

```

Comment:
Is the `gptq_utils` copied from GPTQ? If so, we should claim this in all copied files and add their license to ours.